src/Database/InferenceRequestConfiguration.cs

```diff
@@ -0,0 +1,61 @@
+/*
+ * Apache License, Version 2.0
+ * Copyright 2019-2021 NVIDIA Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Newtonsoft.Json;
+using Nvidia.Clara.DicomAdapter.API.Rest;
+using System;
+using System.Collections.Generic;
+
+namespace Nvidia.Clara.DicomAdapter.Database
+{
+    internal class InferenceRequestConfiguration : IEntityTypeConfiguration<InferenceRequest>
+    {
+        public void Configure(EntityTypeBuilder<InferenceRequest> builder)
+        {
+            builder.HasKey(j => j.InferenceRequestId);
+
+            builder.Property(j => j.TransactionId).IsRequired();
+            builder.Property(j => j.Priority).IsRequired();
+
+            builder.Property(j => j.InputMetadata).HasConversion(
+                        v => JsonConvert.SerializeObject(v, new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore }),
```

Comment:
why allocate `new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore }` so many times, it's a completely reusable type?

Reply:
Fixed 🙏